### ISSUE TITLE
Remove sampleStorageLocation property

### ISSUE THREAD

**Issue by edmondchuc** [ternaustralia/ontology_tern#206]
There has been no evidence of usage for this property.

**Comment by miekeGR** [comment]
Hi @edmondchuc I didn't realise this property existed in the ontology previously. While I do not personally need it right now, I can definitely see its benefits e.g. knowing which storage facility has a particular specimen. But I guess the alternative could be describing this with a more in depth qualifiedAttribution rather than via a geometry. Is that what you do?

**Comment by edmondchuc** [comment]
Yes, it would be useful except the current `sampleStorageLocation` is modelled as a sub-property of `geo:hasGeometry`. So rather than saying which storage facility the specimen is stored at, it's only saying what the location is.

```turtle
tern:sampleStorageLocation
    a rdf:Property ;
    rdfs:label "sample storage location" ;
    rdfs:subPropertyOf <http://www.opengis.net/ont/geosparql#hasGeometry> ;
    skos:definition "A property that links a specimen sample to the location of where it is stored." ;
.
```

If we do keep `sampleStorageLocation` around, it should not be a sub-property of `geo:hasGeometry` and the target nodes may need to be something else (like [schema:Place](https://schema.org/Place)).

I wouldn't suggest using `prov:qualifiedAttribution` as it's only for linking some entity (in this case the sample) to an agent with a role (owner, rights holder, etc).

**Comment by miekeGR** [comment]
Cheers, thanks for that explanation @edmondchuc
